### PR TITLE
chore: test docgen on examples

### PR DESCRIPTION
**Pull request by Artlvns**
chore: introduction of tf-docs for examples.   
docs: README.md docs for project and separate Developer.md for module development

### CODE REVIEW

**Comment by Artlvns** [comment]
This one to be sent to review after actions released  and version tag switched back to v0

File: examples/01-basic/README.md
Comment (Laffs2k5) [inline examples/01-basic/README.md:1]: I suggest we add a header level 1 to these example README files, either manually or automatic as you see fit. Header could be eg. the directory name.

File: examples/02-full/README.md
Comment (Laffs2k5) [inline examples/02-full/README.md:1]: Same comment about header level 1

**Review by Laffs2k5 — APPROVED** [review]
Nice work 🏅

File: examples/01-basic/README.md
Comment (Artlvns) [reply examples/01-basic/README.md:1]: Good point! I didn't noticed that we don't have header now. 🙈 
I'll fix 🫡

**Pull request merged**